### PR TITLE
Specify and update base image patch version

### PR DESCRIPTION
**Pull request by at-wat**

### CODE REVIEW

**Comment by at-wat** [comment]
@f-fl0 please take a look

File: .github/renovate.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["major", "minor"],
+      "enabled": false
```
Comment (f-fl0) [inline .github/renovate.json:8]: Just to be sure I understood correctly, this means do not try to update major or minor version, only patch version, right?

**Review by f-fl0 — APPROVED** [review]
LGTM

File: .github/renovate.json (hunk 1 above)
Comment (at-wat) [reply .github/renovate.json:8]: Yes, major and minor version must be kept same for same Alpine ROS image tag

**Pull request merged**